.github/workflows/release.yml

```
@@ -27,24 +27,31 @@ jobs:
           - os: 'macos'
             target: 'x86_64-apple-darwin'
             artifact: 'libextism.dylib'
+            static-artifact: 'libextism.a'
           - os: 'macos'
             target: 'aarch64-apple-darwin'
             artifact: 'libextism.dylib'
+            static-artifact: 'libextism.a'
           - os: 'ubuntu'
             target: 'aarch64-unknown-linux-gnu'
             artifact: 'libextism.so'
+            static-artifact: 'libextism.a'
           - os: 'ubuntu'
             target: 'aarch64-unknown-linux-musl'
             artifact: 'libextism.so'
+            static-artifact: 'libextism.a'
           - os: 'ubuntu'
             target: 'x86_64-unknown-linux-gnu'
             artifact: 'libextism.so'
+            static-artifact: 'libextism.a'
           - os: 'windows'
             target: 'x86_64-pc-windows-gnu'
             artifact: 'extism.dll'
+            static-artifact: 'libextism.a'
           - os: 'windows'
             target: 'x86_64-pc-windows-msvc'
             artifact: 'extism.dll'
+            static-artifact: 'libextism.lib'
```

I think windows should be `extism.lib`
```suggestion
            static-artifact: 'extism.lib'
```